src/include/abti_mem.h

```diff
@@ -108,26 +104,22 @@ static inline
 ABTI_thread *ABTI_mem_alloc_thread_with_stacksize(size_t *p_stacksize,
                                                   ABTI_thread_attr *p_attr)
 {
-    const size_t header_size = ABTI_MEM_SH_SIZE;
     size_t stacksize, actual_stacksize;
     char *p_blk;
     ABTI_thread *p_thread;
-    ABTI_stack_header *p_sh;
     void *p_stack;
 
     /* Get the stack size */
     stacksize = *p_stacksize;
-    actual_stacksize = stacksize - header_size;
+    actual_stacksize = stacksize - sizeof(ABTI_thread);
```

Comment:
how is this change, and the several other line changes above and below, is related to `ABTI_EXT_STACK`?
It seems memory allocation is made independent of the header size but it's unclear how it relates to `ABTI_EXT_STACK`

Reply:
`header_size` was `sizeof(ABTI_stack_header) + sizeof(ABTI_thread)`.
This `ABTI_stack_header` is only used to store `ABTI_EXT_STACK`. However, it is not needed.
We don't need alias (i.e., `header_size`) any more and just use `sizeof(ABTI_thread)` directly.